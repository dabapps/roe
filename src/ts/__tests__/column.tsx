import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { Column } from '../';

describe('Column', () => {

  it('should match snapshot', () => {
    const tree = renderer.create(
      <Column />
    );

    expect(tree).toMatchSnapshot();
  });

  it('should take regular element attributes', () => {
    const tree = renderer.create(
      <Column className="my-class" />
    );

    expect(tree).toMatchSnapshot();
  });

});
