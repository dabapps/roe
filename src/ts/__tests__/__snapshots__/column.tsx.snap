// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column should match snapshot 1`] = `
<div
  className="column"
/>
`;
